test/integration-test/src/tests/load.rs

```diff
@@ -145,6 +149,24 @@ fn unload_xdp() {
     assert_unloaded("pass");
 }
 
+#[test]
+fn test_loaded_at() {
+    let mut bpf = Bpf::load(crate::TEST).unwrap();
+    let prog: &mut Xdp = bpf.program_mut("pass").unwrap().try_into().unwrap();
+    let t1 = SystemTime::now();
+    prog.load().unwrap();
+    let t2 = SystemTime::now();
+    assert_loaded("pass");
+
+    let loaded_at = prog.info().unwrap().loaded_at();
+
+    assert!(t1 < loaded_at && t2 > loaded_at);
```

Comment:
this assertion doesn't emit information on failure, which seems bad. how about

```rust
let range = t1..t2;
assert!(range.contains(loaded_at), "{range}.contains({loaded_at})");
```

or something like that.

Reply:
Done